app/src/main/res/values/strings.xml

```diff
@@ -2,144 +2,144 @@
-    <string name="Common_Search">Rechercher</string>
-    <string name="Common_Filter">Filtrer</string>
-    <string name="Common_All">Toutes</string>
-
-    <string name="HomePage_Title">Livres de sorts</string>
+    <string name="Common_Close">Close</string>
+    <string name="Common_See_More">See more</string>
+    <string name="Common_Loading_Message">Loading</string>
+    <string name="Common_Validate">Validate</string>
+    <string name="Common_Modify">Modify</string>
+    <string name="Common_Delete">Delete</string>
+    <string name="Common_Search">Search</string>
+    <string name="Common_Filter">Filter</string>
+    <string name="Common_All">All</string>
+    <string name="Common_Level">Level</string>
+    <string name="Common_Leave">Leave</string>
+
+    <string name="HomePage_Title">Spell Books</string>
 
     <!--Spellbooks-->
-    <string name="Spellbooks_Title">Livres de sorts</string>
+    <string name="Spellbooks_Title">Spell Books</string>
 
     <!--Spell Formats-->
-    <string name="spell_book_and_level_format">%1$s Niv. %2$s</string>
+    <string name="spell_book_and_level_format">%1$s Lvl. %2$s</string>
     <string name="spell_action_type_format"><b>Action : </b></string>
     <string name="spell_type_format"><b>Type : </b></string>
-    <string name="spell_effect_format"><b>Effet : </b></string>
+    <string name="spell_effect_format"><b>Effect : </b></string>
 
     <!--Grade-->
     <string name="initial_grade_title">Initial</string>
-    <string name="intermediate_grade_title">Intermédiaire</string>
-    <string name="advanced_grade_title">Avancé</string>
+    <string name="intermediate_grade_title">Intermediate</string>
+    <string name="advanced_grade_title">Advanced</string>
     <string name="arcane_grade_title">Arcane</string>
 
     <string name="spell_grade_int_format">Int : %1$d</string>
-    <string name="spell_grade_zeon_format">%1$d Zéon</string>
-    <string name="spell_grade_retention_format">Maintien : %1$d</string>
-    <string name="spell_grade_retention_daily_format">Maintien : %1$d Quot.</string>
+    <string name="spell_grade_zeon_format">%1$d Zeon</string>
+    <string name="spell_grade_retention_format">Maintenance : %1$d</string>
+    <string name="spell_grade_retention_daily_format">Maintenance : %1$d Daily</string>
 
     <!--Witchspells    -->
-    <string name="Witch_Creation_Title">Liste de sorts</string>
-    <string name="Witchspells_Path_Title_And_Level_Format">%1$s Niv. %2$s</string>
-    <string name="Witchspells_Secondary_Path_Title_And_Level_Format">Voie secondaire : %1$s Niv. %2$s</string>
+    <string name="Witch_Creation_Title">Spell List</string>
+    <string name="Witchspells_Path_Title_And_Level_Format">%1$s Lvl. %2$s</string>
+    <string name="Witchspells_Secondary_Path_Title_And_Level_Format">Secondary path : %1$s Lvl. %2$s</string>
 
-    <string name="Witchspells_Name_Format">Sorts de %1$s</string>
+    <string name="Witchspells_Name_Format">%1$s\'s Spells</string>
```

Comment:
Bonne idée mais si le nom se termine par un "s" ça marche plus :P 
Mets plutôt "Spells of "